wicketd/src/update_tracker.rs

```diff
@@ -1588,6 +1705,43 @@ struct RotInterrogation {
 }
 
 impl RotInterrogation {
+    async fn sp_can_update_bootloader(
+        &self,
+        client: &gateway_client::Client,
+    ) -> bool {
+        let sp_caboose = client
+            .sp_component_caboose_get(
+                self.sp.type_,
+                self.sp.slot,
+                SpComponent::SP_ITSELF.const_as_str(),
+                0,
+            )
+            .await
+            .ok()
+            .map(|v| v.into_inner());
+
+        // Older versions of the SP have a bug that prevents setting
+        // the active slot for the RoT bootloader. Check for these
+        // and skip the update until the SP gets updated
```

Comment:
Just double-checking: we only care about SP version and not RoT version for this?

Reply:
Correct. If the RoT version supports `rot-boot-info` it should support stage0 update.